src/Client/Campaign.php

```diff
@@ -107,29 +108,51 @@ class Campaign extends Common
      * @param string $id
      * @return SFCampaignApiResponse Single Campaign response object as associative array
      * @throws NotFoundException if Campaign with given ID not found
+     * @throws TransferException if connection or request fails twice
+     * @throws \Exception if our loop logic or Guzzle has an unexpexcted case we missed
```

Comment:
not sure its worth declaring that we throw \Exception if there's a bug - I think that's always true and not something we'd ever catch except in a top level, and makes PhpStorm think we should catch it:

![image](https://github.com/user-attachments/assets/714185b6-3915-40ba-b57b-5c8190fee19f)

Reply:
Removed in d8bc2962a99f8d0a71494b29571f27925eb00a16